.github/workflows/test_and_build.yml

```diff
@@ -5,10 +5,16 @@ on:
     branches:
       - main
     types: [opened, reopened, synchronize]
+    paths-ignore:
+      - 'docs/**'
```

Comment:
I already suggested if we name docs to Docs (naming convention wide).

So is this case insensitive?

Reply:
fixed